monorepo-trigger/README.md

```diff
@@ -0,0 +1,36 @@
+---
+title: CI for monorepo
```

Comment:
I suspect there's room for a few monorepo templates so perhaps this should be slightly more specific.